src/extlibs/enlsvg/src/IndirectHeap.cpp

```diff
@@ -5,84 +5,84 @@
-            outList.resize(capacity);
+            key_list.resize(capacity);
+            in_list.resize(capacity);
+            out_list.resize(capacity);
 
-            ticketCheck.clear();
-            ticketCheck.resize(capacity, 0);
-            ticketNumber = 1;
+            ticket_check.clear();
+            ticket_check.resize(capacity, 0);
+            ticket_number = 1;
         }
-        else if (ticketNumber == -1)
+        else if (ticket_number == -1)
         {
-            ticketCheck.clear();
-            ticketCheck.resize(capacity, 0);
-            ticketNumber = 1;
+            ticket_check.clear();
+            ticket_check.resize(capacity, 0);
+            ticket_number = 1;
         }
         else
         {
-            ticketNumber++;
+            ticket_number++;
         }
     }
 
     void IndirectHeap::reinitialise()
     {
-        this->defaultKey = POS_INF;
-        this->heapSize   = 0;
+        this->default_key = POS_INF;
+        this->heap_size   = 0;
 
-        if (ticketNumber == -1)
+        if (ticket_number == -1)
         {
-            std::fill(ticketCheck.begin(), ticketCheck.end(), 0);
-            ticketNumber = 1;
+            std::fill(ticket_check.begin(), ticket_check.end(), 0);
+            ticket_number = 1;
         }
         else
         {
-            ticketNumber++;
+            ticket_number++;
         }
     }
 
-    IndirectHeap::IndirectHeap(int capacity) : heapSize(0)
+    IndirectHeap::IndirectHeap(int capacity) : heap_size(0)
     {
         initialise(capacity, POS_INF);
     }
 
     // Runtime: O(lgn)
-    void IndirectHeap::decreaseKey(int outIndex, double newKey)
+    void IndirectHeap::decreaseKey(int out_index, double new_key)
     {
-        // Assume newKey < old key
-        int inIndex = getIn(outIndex);
+        // Assume new_key < old key
+        int in_index = getIn(out_index);
 
         // Optimisation: Jump the newly set value to the bottom of the heap.
         // Faster if there are a lot of POSITIVE_INFINITY values.
         // This is equivalent to an insert operation.
-        if (getKey(inIndex) == POS_INF)
+        if (getKey(in_index) == POS_INF)
         {
-            swapData(inIndex, heapSize);
-            inIndex = heapSize;
-            ++heapSize;
+            swapData(in_index, heap_size);
+            in_index = heap_size;
+            ++heap_size;
         }
-        setKey(inIndex, newKey);
+        setKey(in_index, new_key);
 
-        bubbleUp(inIndex);
+        bubbleUp(in_index);
     }
 
 
     // Runtime: O(lgn)
     // returns index of min element
     int IndirectHeap::popMinIndex()
     {
-        if (heapSize == 0)
+        if (heap_size == 0)
             std::cout << "ERROR: EMPTY HEAP!" << std::endl;
-        else if (heapSize == 1)
+        else if (heap_size == 1)
         {
-            --heapSize;
+            --heap_size;
             return getOut(0);
         }
         // nodeList.size() > 1
```

Comment:
I know that this is not part of this ticket, but could you also remove the commented out code here?

Reply:
done